File: Linguagem de Programação - I/Aula 02/exercicio04.c
#include <stdio.h>
#include <stdlib.h>

int main()
{

int segundos_do_evento;
int horas;
int minutos;
int segundos;

printf ("Digite a duracao do evento em segundos.");
scanf ("%d", &segundos_do_evento);

horas = segundos_do_evento/3600;
minutos = segundos_do_evento/60;
segundos = segundos_do_evento;

printf ("O evento durou o equivalente a %d horas, %d minutos ou %d segundos.",horas , minutos, segundos );

}

File: Linguagem de Programação - II/Aula 01/exercicio04.py
tempo_de_duracao = int(input("Digite o tempo de duração do evento em segundos: "))

horas = tempo_de_duracao / 3600
minutos = tempo_de_duracao / 60
segundos = tempo_de_duracao

'''arredondamento'''
horas = round (horas, 2)
minutos = round (minutos, 2)

print (f"O evento durou {horas} horas; {minutos} minutos e {segundos} segundos")